python_modules/dagster/dagster/_core/execution/run_metrics_thread.py

```diff
@@ -211,7 +223,7 @@ def _capture_metrics(
 
         except:
             if logger:
-                logger.error("Exception during capture of metrics, will cease capturing")
+                logger.error("Exception caught during capture of metrics, will cease capturing")
```

Comment:
logger.exception will include the stack trace, which seems useful here rather than swallowing it?